.github/dependabot.yml

```diff
@@ -23,6 +24,7 @@ updates:
       - "/tests/regression"
     schedule:
       interval: "daily"
+    open-pull-requests-limit: 10
```

Comment:
Should we increase the limit on github actions/rust tools? This could technically result in 30 open PRs I think, and I'm not sure we care as much about the non-rust bindings updates. Maybe we should only increase the limit for /standard and /extended?